src/dndbeyond/content-scripts/character.js

```diff
@@ -124,6 +124,7 @@ function rollItem(force_display = false) {
     //console.log("Properties are : " + String(properties));
     const item_name = $(".ct-item-pane .ct-sidebar__heading .ct-item-name,.ct-item-pane .ct-sidebar__heading .ddbc-item-name")[0].firstChild.textContent;
     const item_type = $(".ct-item-detail__intro").text();
+    const item_tags = $(".ct-item-detail__tags-list").find(".ct-item-detail__tag").map(function(){ return $(this).text().toLowerCase(); }).toArray();
```

Comment:
`$(".ct-item-detail__tags-list").find(".ct-item-detail__tag")` should be `$(".ct-item-detail__tags-list .ct-item-detail__tag")`

`.map(function(){ return $(this).text().toLowerCase(); }).toArray()`  should be `.toArray().map(el => el.textContent)`
Didn't realize we can `.map` on the jquery object itself, but since you're doing a `toArray` anyways, might as well do it first. Also, I don't see a reason to lowercase the tags, or to use jquery to just get the text content.
This also means the check in the roll renderer should check for the non-lowercase version.
Smart idea to send the tags, I thought I'd have to keep a list of hardcoded instrument names

Reply:
Good points! I've now implemented these suggestions.

and ty! :P